recipes/wbgapi/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "wbgapi" %}
+{% set version = "1.0.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: cd04c75830656ec4fb08b5061e06da9ac08b210e2f5add8217a173ef05223ba0
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.0
+    - pip
+    - requests
+    - PyYAML
+    - tabulate
+  run:
+    - python >=3.0
+    - requests
+    - PyYAML
+    - tabulate
+
+test:
+  imports:
+    - wbgapi
```

```suggestion
    - wbgapi
  commands:
    - pip check
  requires:
    - pip
```

This checks that the dependencies are correctly added.